### DISCUSSION TITLE
Already used!!

### DISCUSSION THREAD

**Discussion by FardinIR** [outline/outline#4576] [Self Hosting] [answered]
Hello, I have a problem, how should I fix this error or get a new address?
![IMG_20221214_183358_632](https://user-images.githubusercontent.com/120580683/207631790-de7e689c-6623-4082-9a13-36872fe6d1cb.jpg)

**Comment by almereyda** [comment, marked as answer]
The error message that you are presenting is in no way familiar to what the software project served here provides. You are probably talking about

- https://getoutline.org/

who have their issue trackers at

- https://github.com/Jigsaw-Code/?q=outline

Good luck securing your internet access. I would suggest to raise your question over there.

Also have a look at their support portal for troubleshooting aids about your problem and their Reddit community, where people might be able to help out:

- https://support.getoutline.org/
- https://www.reddit.com/r/outlinevpn/

**Reply by FardinIR** [reply]
thanks

**Reply by almereyda** [reply]
If you have a little more time, please also do mark my answer as *accepted*, in so that this discussion can be cleanly concluded for the others reading here with us.
